processor.go

```diff
@@ -45,33 +45,36 @@ func NewProcessor(net Network, graph Graph, build Builder, strat Strategy, sign
 		sign:   sign,
 		verify: verify,
 		cache:  cache,
-	}
+	} //  processor instantiation; fields can be set to anything that implements the corresponding interfaces
 
 	return &pro
 }
 
 func (pro *Processor) Bootstrap() error {
 
 	// get current tip of the state
-	tip, err := pro.graph.Tip()
+	tip, err := pro.graph.Tip() // obtain the tip of the DAG upon which to build
+	// in this function we want to bootstrap from genesis
```

Comment:
Not necessarily a DAG. The point is that the graph can be anything, and the consensus algorithm in this case doesn't really care.